process/process.lua

```diff
@@ -250,7 +250,7 @@ end
 -- @tparam {table} env The environment to initialize the state with
 local function initializeState(msg, env)
   if not Seeded then
-    chance.seed(tonumber(msg['Block-Height'] .. stringToSeed(msg.Owner .. msg.Module .. msg.Id)))
+    chance.seed(tonumber(msg['Block-Height'] .. stringToSeed(msg.Owner .. (msg.Module or "") .. msg.Id)))
```

Comment:
a tag needs to be provided when creating the process (we get an error when it is not set)

Reply:
cc @atticusofsparta

Reply:
fixed in - [f25a9bc](https://github.com/ar-io/aos/pull/2/commits/f25a9bcdde8794191287685baeea6bfca6f659b5)